packages/build-info/src/frameworks/solid-start.ts

```diff
@@ -3,23 +3,23 @@ import { BaseFramework, Category, Framework } from './framework.js'
 export class SolidStart extends BaseFramework implements Framework {
   readonly id = 'solid-start'
   name = 'Solid Start'
-  npmDependencies = ['solid-start']
+  npmDependencies = ['@solidjs/start']
```

Comment:
Do we have currently any `solid-start` usage? Because if we do - by removing `solid-start` from here we might change some behaviors?

And if we would have to keep `solid-start` matching - then also commands/directories would also need to keep working for it?

Reply:
afaik, we only do framework detection on site creation? so that means this will only affect new sites and since the`solid-start` package is deprecated, I think this is fine?

Reply:
Nope this is used in the CLI as well and could break existing flows with the CLI using `ntl dev` or `ntl build`

Reply:
I think you must detect if it's old `solid-start` or vinxi based